package.json

```diff
@@ -35,18 +35,18 @@
   },
   "homepage": "https://github.com/dabapps/eslint-config-dabapps",
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.15.2",
-    "@typescript-eslint/parser": "^4.15.2",
```

Comment:
Why have you uninstalled the typescript parser?

Reply:
Probably forgot to put it back, reverted 👍